README.md

```diff
@@ -797,6 +797,22 @@ params do
 end
 ```
 
+By default Grape 1.x presents the parameters to the endpoint as an
+ActiveSupport::HashWithIndifferentAccess. This is a change from 0.x where a
+Hashie::Mash was previously presented.
```

Comment:
This and the restore note belongs in UPGRADING, not README. Lets keep it consistent.